rascaline/src/calculators/radial_basis/mod.rs

```diff
@@ -10,8 +10,8 @@ pub use self::tabulated::SplinePoint;
 pub enum RadialBasis {
     /// Use a radial basis similar to Gaussian-Type Orbitals.
     ///
-    /// The basis is defined as `R_n(r) ∝ r^n e^{- r^2 / (2 σ_n^2)}`, where `σ_n
-    /// = cutoff * \sqrt{n} / n_max`
+    /// The basis is defined as $R_n(r) \propto r^n \exp\left[ -r^2 / (2 σ_n^2) \right]$,
```

Comment:
This is not rendered as math, so I think the previous code is better here: https://rascaline--204.org.readthedocs.build/en/204/references/calculators/spherical-expansion.html#json-schema-radialbasis

Reply:
i only checked the rust docs and there it looks fine. Sure I can adjust this.

Reply:
yeah, this is used in two places. We could (and probably should) improve the calculator reference to render math, but the code extracting the doc is a bit crude.

Reply:
Yeah its fine. Now I know :-)